src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -2,6 +2,10 @@
 
 import java.util.HashSet;
 import java.util.Set;
+import java.lang.*;
```

Comment:
This is a violation of coding styles. The exact class/interface need to be specified.